skypy/utils/tests/test_completeness_function.py

```diff
@@ -0,0 +1,37 @@
+import numpy as np
+
+
+def test_logistic_completeness_function():
+    from skypy.utils.completeness_function import logistic_completeness_function
+
+    # Test that (nb, nm) array is returned if m in (mb, nm) and m95 and m50 in (nb, ) are given
+    m = np.ndarray((5, 50))
```

Comment:
Is this for 5 galaxies with 50 values of magnitudes?

Reply:
also use unique variable names for each test block. You could use here `m_ndarray`

Reply:
That's is 50 values of magnitudes for 5 cases with different completeness values.

I personally dislike giving unique variable names. I prefer redefining the parameters to save memory.